netket/experimental/driver/tdvp.py

```diff
@@ -54,7 +54,8 @@ def __init__(
         self,
         operator: AbstractOperator,
         variational_state: VariationalState,
-        integrator: AbstractSolver,
+        solver: AbstractSolver = None,
+        integrator: AbstractSolver = None,
         *,
         t0: float = 0.0,
         propagation_type: str = "real",
```

Comment:
```suggestion
        solver: AbstractSolver = None,
        *,
        t0: float = 0.0,
        propagation_type: str = "real",
        integrator: AbstractSolver = None,
```

this can become a keyword argument. If its passed positionally the name is not specified.